### ISSUE TITLE
Implement initial jump behavior (animated)

### ISSUE THREAD

**Issue by emily-phet** [phetsims/faradays-law#103]
To support JH's design process, would be helpful to have working version of his reflected horizontal jumping key idea. Magnet would need to animate to the reflected position. JH's design doc here:

https://wiki.fluidproject.org/display/fluid/PhET+Faraday%27s+Law+Keyboard+Interaction+Sketch

**Comment by emily-phet** [comment]
not sure how challenging this would be - @jessegreenberg thoughts?

**Comment by jessegreenberg** [comment]
This seems pretty straight forward. @mbarlow12 and I discussed, he is going to work on this in a one-off branch, thanks!

**Comment by mbarlow12** [comment]
Noting a couple considerations as we move forward with this:
- will the animation be interruptible? If so, what behavior is desired when that happens?
- can augment keys alter the animation once the 'j' key is released?
    - applies to both pressing or releasing an augment key

**Comment by mbarlow12** [comment]
Another thought is that 'j' could be a modifier key so that j + left arrow would actually activate the animation instead of releasing the 'j' key. This comes from the quick that, currently, when the 'j' key is pressed, the user is essentially locked into a course of action, and that might be something we want to avoid.

**Comment by emily-phet** [comment]
emailing Jon Hung to see if he wants to weigh in on this.

**Comment by mbarlow12** [comment]
Notes from IDRC Meeting:
- allow interrupts to the moving/animating magnet (possibly space or esc).
- default drag movement similar/identical to Coulomb's Law
- potential to add speed modifiers (e.g. 1,2, keys while moving to accelerate)

**Comment by mbarlow12** [comment]
The initial behavior has been implemented in https://github.com/phetsims/faradays-law/commit/83e6809d920ffc4e7efd06e30175d56f786d4fc2. Link below:

https://www.colorado.edu/physics/phet/dev/html/faradays-law/1.4.0-keyboard_jump.3/phet/faradays-law_en_phet.html?accessibility

As of right now, the `SHIFT` key slows the animation as that aligns more closely with other a11y interactions. The respective speeds are easy to manipulate as well.

A few notes:
- The user can hold the 'j' key down and observe its destination dynamically while pressing WASD or arrow keys
- Any key press will interrupt the animation
- The magnet jumps across the vertical line of the sim. Should the reflection take place around the center of the coil(s) instead?
- Consider tweaking how the focus highlight & arrows are handled (e.g. remove arrows after the first few key interactions)
- Should the magnet animation speed be based on its distance away from the center (farther=faster?

**Comment by mbarlow12** [comment]
Notes from Friday's IDRC meeting:
- ensure that the magnet cannot pass through the coils
- alter animation speed dynamically with `shift` key press
- add increased speed modifier
- remove arrows after initial keyboard interactions

**Comment by mbarlow12** [comment]
New jumping behavior implemented in:
https://phet-dev.colorado.edu/html/faradays-law/1.4.0-jump.1/phet/faradays-law_en_phet.html

**Comment by emily-phet** [comment]
Assigning to @terracoda - she'll probably recall the desired behavior better than I will.

**Comment by jbphet** [comment]
The a11y efforts for this sim stalled out for a while, and were recently revived (see https://github.com/phetsims/faradays-law/issues/174).  The sim currently has some custom drag handling capabilities where the user can use the 1, 2, and 3 keys to move the magnet horizontally at three different speeds across the screen.  I'm in the midst of adding the space-bar-to-pick-up functionality, and my plan is to preserve the 1-2-3 horizontal movement feature as well.  @emily-phet and @terracoda - please let me know if I should _not_ be trying to preserve this functionality.  It seems pretty cool to me, but it would certainly be easier development-wise to take it out if we don't actually need it in the final product.

**Comment by emily-phet** [comment]
Yes, let's keep this. @jbphet please check in with @terracoda re keyboard shortcut dialog if this wasn't included already.

**Comment by terracoda** [comment]
Yes, I will need to add special instructions to the Keyboard Shortcuts dialog. I do not recall exactly from memory how it works, but I am sure I can figure it out by reviewing this issue. Keeping myself assigned.

**Comment by jbphet** [comment]
The "jump behavior" code has been revived and is working reasonably well, and the behavior is now described in the Keyboard Help dialog.  I think I'll close this issue, and any other work on this feature and the help documentation that supports it will be done under #174 and #176.
